variables.tf

```diff
@@ -72,6 +72,21 @@ variable "include_self_ingress_rule" {
 }
 
 variable "allow_major_version_upgrade" {
-  description = "hether or not to allow major version upgrades"
+  description = "Whether or not to allow major version upgrades"
   default = "no"
 }
+
+variable "auto_minor_version_upgrade" {
+  description = "Whether or not to enable auto minor version upgrades"
+  default = "no"
```

Comment:
This represents a breaking change although since you're experiencing downtime because of this, I think it makes sense. Could you update the CHANGELOG with details of the breaking change, under a section for the next major version of the module?

Reply:
I can set default to "yes" to not cause a breaking change. Downtime is barely observable (a few seconds to restart the db).

Reply:
I'm ok with the breaking change but like to signal it via a major version number change and a CHANGELOG entry.
Up to you whether you change to "yes" or update CHANGELOG, happy either way :)

Reply:
Done. Default for auto_minor_version_upgrade is "yes".